public/data/productData.json

```
@@ -0,0 +1,49 @@
+{
+  "results": [
+    {
+      "name": "(남성) 프렌치 레귤러핏 반팔 폴로",
+      "price": "139000",
```

```suggestion
      "price": 139000,
```
number로 받으셔야 나중에 계산을 하거나 할 때 수월하지 않을까요?

---

수정하겠습니다 !